lib/fog/kubevirt/compute/requests/create_vm.rb

```diff
@@ -3,6 +3,9 @@ module Kubevirt
     class Compute
       class Real
         def create_vm(vm)
+          addr = kubevirt_client.rest_client
```

Comment:
I have a doubt about this: we provide the version for kubeclient to create the client, and then we query it for its version.

The other option I can see is using the client wrapper (ExceptionWrapper) to maintain also the version used for creating this client.

Reply:
There was no way to access it from the client. I will explore different ways.